### PR TITLE
Add command `trace`

### PR DESCRIPTION
**Pull request by kezhenxu94**
### Motivation

Add command `trace` to query trace by trace ID and visualize the traces

### Modification

- Add command `trace` to query and visualize traces.

- Move `main.go` to `cmd`, as per the recommendation of GO CLI package structure.

- Move GraphQL query to individual `*.graphql` file, for more friendly development, with IDE plugin.

### Result

- Traces can be queried and visualized by `swctl trace <trace id>`

- GraphQL queries can be debugged in IDE, with some plugins

- Closes https://github.com/apache/skywalking/issues/3900

### CODE REVIEW

File: .graphqlconfig [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "SkyWalking GraphQL Schema",
+  "extensions": {
+    "endpoints": {
+      "SkyWalking GraphQL Endpoint": {
+        "url": "http://122.112.182.72:8080/graphql",
```
Comment (wu-sheng) [inline .graphqlconfig]: I think this is an IP leak.

Comment (kezhenxu94) [reply .graphqlconfig]: > I think this is an IP leak.

What do you mean by saying "leak", it's a public demo address

Comment (kezhenxu94) [reply .graphqlconfig]: Everyone who visits the main repo can get access to this address

**Comment by kezhenxu94** [comment]
image

File: .graphqlconfig (hunk 1 above)
Comment (kezhenxu94) [reply .graphqlconfig]: Anyway, fixed

**Comment by wu-sheng** [comment]
Cool, do you create an interactive tree? Could you post a youtube or jpg including the interaction, this is a good twitter and documentation material.

**Comment by wu-sheng** [comment]
Any new lib added for this?

**Comment by kezhenxu94** [comment]
> Cool, do you create an interactive tree? Could you post a youtube or jpg including the interaction, this is a good twitter and documentation material.

I can create one, soon

> Any new lib added for this?

No new lib for the trace tree, but a new lib for bundling the `*.graphql` files, which is `packr`, and I have added it into the `LICENSE` file, there's no `NOTICE` file in the `packr` project

**Comment by kezhenxu94** [comment]
[asciicast](https://asciinema.org/a/hmnVVAKQuRhPRrDOGxHFmroMC)

**Pull request merged**